news/san_francisco.py

```diff
@@ -33,34 +34,29 @@ class SanFranciscoNews(NewsScraper):
         home_page_url='https://sf.gov/news/topics/794'
     )
 
-    START_URL = 'https://sf.gov/news/topics/794'
+    URL = 'https://sf.gov/news/topics/794'
```

Comment:
Yeah, this is what I’d initially intended to do, but *the very first feedback I got* from @kengoy was that we were already surfacing too much/going back too far in time (See #43). :P

So I more or less dropped the idea of ever iterating through multiple pages.

> it looks like the Santa Clara scraper is grabbing all news items (earliest item is January 31, 2020).

It is, but only because the first page of results for Santa Clara shows the whole year to date. If we wanted a more “complete” set, we’d still have to do paging for Santa Clara, too. But in practice we never will, because we probably only want the last couple weeks of news.

> Perhaps we need to decide on a standard rolling window to grab news for all counties, such as just news items from the past 2 weeks.

Yeah. I think we’ll probably pick some default when #43 is implemented.